democracy/migrations/0012_auto_20160502_1219.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use the `-n` flag with `makemigrations` to specify the migration's name. (`auto` migration names are not very nice.)